bindings_ffi/src/mls.rs

```diff
@@ -120,7 +123,7 @@ pub async fn create_client(
     account_identifier: FfiIdentifier,
     nonce: u64,
     legacy_signed_private_key_proto: Option<Vec<u8>>,
-    history_sync_url: Option<String>,
+    device_sync_server_url: Option<String>,
```

Comment:
We probably don't need to change the low level but on the SDK side and the bindings we should call this `history_transfer_server_url` can be done in a follow up but just noting.